src/consensus/params.h

```diff
@@ -80,6 +80,13 @@ struct Params {
     int64_t DifficultyAdjustmentInterval() const { return nPowTargetTimespan / nPowTargetSpacing; }
     uint256 nMinimumChainWork;
     uint256 defaultAssumeValid;
+
+    /**
+     * If true, witness commitments contain a payload equal to a Bitcoin Script solution
+     * to the signet challenge.
+     */
+    bool signet_blocks{false};
```

Comment:
359badf294c8fa632b7df479a3e146cf28658f6b

nit, just noting `signet_blocks` is only used in the next commit.